tools/wltests/platforms/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.img
```

Comment:
Mmm... not entirely convinced about defining that for all platforms... moreover, for wahoo we already use the `ARTIFACTS_PATH`:
   https://github.com/ARM-software/lisa/blob/master/tools/wltests/helpers#L32

To collect all the platform generates assets, e.g.:
https://github.com/ARM-software/lisa/blob/master/tools/wltests/platforms/wahoo_android_v4.4/build_images#L64

Maybe you should add something similar for the XZ2?

Reply:
Right, I didn't see that ``ARTIFACTS_PATH` thing, I'll add it in.
As for the gitignore, why aren't you convinced about it ? We shouldn't ever want to commit build artifacts, and if we eventually do we can -f them.